Dockerfile

```diff
@@ -11,14 +11,12 @@
 
 WORKDIR /app
 
-ENV PYTHONUNBUFFERED=1
+ENV PYTHONUNBUFFERED="default"
+ENV ENV_ENGINE="default"
+ENV ENV_NAME="default"
+ENV ENV_USER="default"
+ENV ENV_PASSWORD="default"
+ENV ENV_HOST="default"
+ENV ENV_PORT="default"
```

Comment:
The `ENV_PORT` variable is set to 'default'. Ensure this value is appropriate for your application or provide instructions on how it should be overridden during deployment.